test/e2e/framework/addon/chart/addon.go

```diff
@@ -189,8 +201,21 @@ func (c *Chart) getHelmVersion() (string, error) {
 
 // Deprovision the deployed instance of tiller-deploy
 func (c *Chart) Deprovision() error {
-	err := c.buildHelmCmd("delete", "--namespace", c.Namespace, c.ReleaseName).Run()
+	cmd := c.buildHelmCmd("delete", "--namespace", c.Namespace, c.ReleaseName)
+	cmd.Stdout = nil
+	out, err := cmd.StdoutPipe()
 	if err != nil {
+		return err
+	}
+	defer out.Close()
+
+	err = cmd.Run()
+	if err != nil {
+		_, err := io.Copy(os.Stdout, out)
+		if err != nil {
+			return err
```

Comment:
issue (blocking): this suppresses the actual error if Copy returns an error

suggestion: return fmt.Errorf("cmd.Run: %v: io.Copy: %v", err, err2) or similar